terraform/P2/scripts/mongodb/setup_replication.sh

```diff
@@ -2,17 +2,47 @@
 
 # File to be called in the primary node only
 
-# Get the machine's IP address dynamically
-MONGO_HOST_1=${machine_ip_1}
-MONGO_HOST_2=${machine_ip_2}
-MONGO_HOST_3=${machine_ip_3}
+# Check if required arguments are provided
+if [ "$#" -ne 3 ]; then
+    echo "Usage: $0 <machine_ip_1> <machine_ip_2> <db_port>"
+    exit 1
+fi
+
+# Wait for the MongoDB service to start
+echo "Waiting for the MongoDB service to start..."
+sleep 30
```

Comment:
```suggestion
sleep 30
```
Le 30 dans une var pour que ce soit plus parlant + plus facile a changer si jamais il faut plus de temps ?

Reply:
Je pense pas que ca soit nécessaire a la limite on peux update le commentaire pour que cela soit plus clair